### ISSUE TITLE
Delete Moved / Removed files from jellyfin database

### ISSUE THREAD

**Issue by hasezoey** [jellyfin/jellyfin#6704]
**Describe the bug**
i had a temporary storage directory in my music folder structure, and it just happened that it was long enough there that jellyfin parsed it, but now i cannot find something to remove all missing data albums / songs

**System (please complete the following information):**
 - OS: Ubuntu Server 21.04 (Jellyfin server), Manjaro 21.1.5 Firefox (Client)
 - Virtualization: Docker
 - Clients: Browser
 - Browser: Firefox 93
 - Jellyfin Version: Jellyfin Docker 10.8.0 (`jellyfin/jellyfin:unstable@sha256:a2fcaeef196b3f27b3e93600776516b6e68b89fc9dabf161c2e6c08eeac23acb`)
 - FFmpeg Version: Whatever is in the docker by default
 - Playback: Not applicable to this issue
 - Hardware Acceleration: Not applicable to this issue
 - Installed Plugins: MusicBrainz
 - Reverse Proxy: nginx
 - Base URL: yes, Not applicable to this issue
 - Networking: Docker (Host)
 - Storage: Local (Docker bind mount)

**To Reproduce**
- have a music library set-up
- create a temporary directory inside the music library
- have songs in that temporary directory
- let jellyfin fetch everything (either by itself, or by force with "Scan Library" -> "Scan for new and updated files")
- verify that jellyfin parsed the songs / albums
- move / remove the temporary directory

**Expected behavior**
Expected there to be either a icon (or text) there on albums / songs noting that the files are missing (and not just when trying to play the file)
and / or having a button somewhere to remove all missing files

**Server Logs**

**FFmpeg Logs**
Not applicable to this issue

**Browser Console Logs**
Not applicable to this issue

**Screenshots**
Not applicable to this issue

**Additional context**
i already tried "Scan All Libraries" (see #6703) and on the music library individually all 3 scan methods
Edit: i forgot to mention i also already tried `Clean Cache Directory` and `Optimize Database`
note: yes i can remove the duplicate songs / albums individually, but i am asking if there is / request to have something that takes care of it in bulk (like when running `Scan for new and updated files`)

**Comment by hasezoey** [comment]
Update:
some time has passed since the creation of this issue (the day i noticed i had this problem), and i just now noticed that jellyfin somehow cleaned-up all the removed albums / songs, they are no longer there (which didnt happen when manually scanning the libraries)

current jellyfin in use: `jellyfin/jellyfin:unstable@sha256:338eb8a48fc2a8b7e4327745a525cddac1200b101ba96a78300855ef45fc72d0`

**Comment by Bond-009** [comment]
A rescan should do it

**Comment by hasezoey** [comment]
> A rescan should do it

since when? because when i first created this issue i had tried all re-scan option on a per-library bases and on a more global bases (through admin panel instead of library list), but the deleted ones did not disappear, and only after some time (probably through some automated process, that could not be manually triggered) they were removed

**Comment by Bond-009** [comment]
Since always, that's why this issue exists: #1714

**Comment by pdf** [comment]
@Bond-009 I have the same problem with (at least one) album that simply will not remove itself from the library after retagging and reorganising some sections of my library. The files definitely don't exist on the filesystem, but no amount of rescanning will remove them from the library.

Maybe of interest, every time I scan it says `Library Sync: Updated 2 items` though no changes have occurred on the filesystem, and no changes appear to be propagated to the DB.

I tried enabling debug logging per the [Troubleshooting](https://jellyfin.org/docs/general/administration/troubleshooting/#debug-logging) page, but do not see any debug logs.

Jellyfin 10.8.8-1 from the Jellyfin Ubuntu repo.
